### ISSUE TITLE
[BUG] InsecureDirectBank does not display bank balance correctly

### ISSUE THREAD

**Issue by ismisepaul** [OWASP/SecurityShepherd#624]
A float is used to store the value of `10000000000` from the database

```
+-----------------+
| account_balance |
+-----------------+
|     10000000000 |
+-----------------+
```

When returned to the user the following is presented `1.0E10`

```
HTTP/1.1 200 
Content-Length: 6
Date: Fri, 15 Apr 2022 17:16:11 GMT
Connection: close

1.0E10
```

Instead of a float a long should be used.

**Comment by ismisepaul** [comment]
Also fixing this weird display seen when running docker
![key-not-displaying-properly-directobjectbank](https://user-images.githubusercontent.com/912403/163633856-1ce7e57f-971d-4435-89c0-3961cbe8b08f.png)

**Comment by ismisepaul** [comment]
fixed in #638
